src/Components/Projects/Projects.tsx

```diff
@@ -1,6 +1,99 @@
 
+import { useContext, useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { AuthContext } from "../../Context/AuthContext";
+import axios from "axios";
+import NoData from "../../Shared/NoData/NoData";
+
+
 export default function Projects() {
+
+  const navigate = useNavigate();
+  const {baseUrl , requstHeaders} : any = useContext(AuthContext);
+  const [projectList , setProjectList] = useState([]);
+  const [isLoding , setIsLoding] =useState(false);
+
+
+    // *************** to get all projects *****************
+    const getCategoryList = ()=>{
+
+      setIsLoding(true)
+      axios.get(`${baseUrl}/Project/manager` , 
+      {
+        headers: requstHeaders ,
+      })
+      .then((response)=>{
+        console.log(response.data);
+        setProjectList(response?.data.data)
+        setIsLoding(false)
+      
+      }).catch((error)=>{
+        // error(error?.response?.data?.message || "Not Found Categorys")
+        console.log(error.response.data.message);
+        setIsLoding(false)
+      })
+    }
+
+    //**************** for navigate to add new project ****************** 
+    const addNewProject = ()=> {
+      navigate("/dashboard/addProject")
+    }
+
+    useEffect( ()=> {
+      getCategoryList()
+    } , [])
+
+
   return (
-    <div>Projects</div>
+    <>
+      
+      {/* **************** to content above table ****************** */}
+      <div className='bg-white header d-flex justify-content-between px-4 py-3 '>
+          <h3> Projects </h3>
+          <button onClick={addNewProject} className="shredBtn" > <i className="fa fa-plus"></i> Add New Project </button>
```

Comment:
This button doesn't have a hover state (having a lighter color of the current one for example)